staff/models.py

```diff
@@ -135,12 +135,16 @@ def get_current(**kwargs):
         return Person.objects.filter(ophase=Ophase.current(), **kwargs)
 
     @staticmethod
-    def get_by_email_address(address):
+    def get_by_email_address(address, ophase):
         try:
-            return Person.objects.get(email__iexact=address)
+            return Person.objects.get(email__iexact=address, ophase=ophase)
         except Person.DoesNotExist:
             return None
 
+    @staticmethod
+    def get_by_email_address_current(address):
```

Comment:
isn't this an example of where optional parameters make sense? either `ophase=Ophase.current()` works or `ophase=None` should do the trick when dispatching manually in the body of the function (of course, this is semantically a bit abusive but allows to get rid of the additional function).

But it's up to you, if you prefer two separate methods, keep it this way.